README.rst

```diff
@@ -52,7 +112,7 @@ Since ``ExplainCollection`` instances provide all the same methods provided by `
 
 
 Explaining commands in a script
```

Comment:
The explain output for ``update_one`` above looks oddly formatted. Can you fix it?

Reply:
Done.